recipes/libgd/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: libgd
-  version: 2.1.1
+  version: 2.1.1post
```

Comment:
can you use bioconda instead of post? This should clarify that the package represents the bioconda build.